spec/tile_bag_spec.rb

```diff
@@ -0,0 +1,56 @@
+require "spec_helper"
+require './lib/tile_bag.rb'
+
+describe TileBag do
+  before :each do
+    @tile_barf = TileBag.new
+  end
+
+  describe "#new" do
+    it "create a new instance of Player" do
+      expect(@tile_barf).to be_an_instance_of TileBag
+    end
+
+    it "expect there to be the correct number of letters" do
+      expect(@tile_barf.tile_hash[:J]).to eq 1
+    end
+  end
+
+  describe "#make_tiles" do
+    it "creates and array of tiles with 'A' in the first position" do
+      expect(@tile_barf.tile_bag[0]).to eq "A"
+    end
+
+    it "ensures the last index of tile_bag is z" do
+      expect(@tile_barf.tile_bag[-1]).to eq "Z"
+    end
+
+    it "creates an array that is the correct length" do
```

Comment:
Nice! I think this is preferable to checking if `@tile_barf.tile_bag.length` is equal to 98, as it allows for flexibility. If you changed the default starting tiles, this test would still work and still properly check that every new `TileBag` instance  started with the correct number of tiles.